frontend/src/components/pages/UserProfilePage.tsx

```diff
@@ -270,24 +262,18 @@ const UserProfilePage = () => {
             Email
           </Heading>
           <Text>{email}</Text>
-          <Heading size="sm" marginTop="36px">
-            Last active on
-          </Heading>
-          <Text>TODO</Text>
           {isAdmin && resumeId && (
             <>
               <Heading size="sm" marginTop="36px">
                 User Files
               </Heading>
-              {/* TODO: replace file name */}
               <Button variant="link" onClick={() => downloadFile(resumeId)}>
-                resume.pdf
+                {`${resumeFileName}.pdf`}
```

Comment:
something i'm not sure about: if the user uploads their resume in `.docx` or some other format, the displayed name will be wrong (the file will still be downloaded correctly tho)

it's not that straightforward to get the file extension from the user profile page, so just curious how we should deal with this

Reply:
down to ommit .pdf from the name 👍